src/studio/src/designer/backend/Designer.csproj

```diff
@@ -6,10 +6,9 @@
     <AssemblyName>Altinn.Studio.Designer</AssemblyName>
     <Company>Altinn</Company>
     <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
-    <TypeScriptToolsVersion>3.7</TypeScriptToolsVersion>
+    <TypeScriptToolsVersion>Latest</TypeScriptToolsVersion>
```

Comment:
What does this do? (just my curiosity, not really questioning the change)

Reply:
In our case nothing, because we don't use TypeScript in the solution. It gave a warning on because it was a mismatch between the installed version and the one specified. But I could probably remove it all together, it came by default with the project type I think.

Reply:
Leaving it for now as there are more settings probably not relevant that could be removed, but need to look more into the consequences.